setup/productionize/caching/externalize-redis.md

```diff
@@ -38,6 +38,14 @@ overrideBaseUrl: $REDIS_ENDPOINT
 skipLifeCycleManagement: true
 ```
 
+Additionally, we will create the following file in order to customize the gate service. `~/.hal/$DEPLOYMENT/profiles/gate-local.yml`:
```

Comment:
nit: replace "we will create..." with "you'll need to create..."

Reply:
or just: "Create..."